### ISSUE TITLE
Android opnes just date picker in datetime mode

### ISSUE THREAD

**Issue by AviShmueli** [skwasjer/skwas-cordova-plugin-datetimepicker#10]
I am trying to use this plugin to both date and time picker.
But only the date picker opening.

my code:
```
var showDatePicker = function () {
            var deferred = $q.defer();

            document.addEventListener("deviceready", function () {
                cordova.plugins.DateTimePicker.show({
                    mode: "datetime",
                    date: new Date(),
                    allowOldDates: false,
                    allowFutureDates: true,
                    minuteInterval: 15,
                    locale: "EN",
                    okText: "OK",
                    cancelText: "Cancel"
                }, function(newDate) {
                    deferred.resolve(newDate);
                    }, function (err) {
                        // Handle error.
                        console.error(err);
                    });
            }, false);

   return deferred.promise;
        }
```

Thanks..

**Comment by skwasjer** [comment]
This has to do with android material design. We need some changes in the code and a few style declarations (one for date and one for datetime), which can then be referenced by code. I will have to take a look

**Comment by AviShmueli** [comment]
Thanks..
In the meanwhile i am using this solution:

```
document.addEventListener("deviceready", function () {
                cordova.plugins.DateTimePicker.show({
                    mode: "date",
                    date: new Date(),
                    allowOldDates: false,
                    allowFutureDates: true,
                    minuteInterval: 15,
                    locale: "EN",
                    okText: "ok",
                    cancelText: "cancel"
                }, function(newDate) {
                    cordova.plugins.DateTimePicker.show({
                        mode: "time",
                        date: new Date(),
                        allowOldDates: false,
                        allowFutureDates: true,
                        minuteInterval: 15,
                        locale: "EN",
                        okText: "ok",
                        cancelText: "cancel"
                    }, function (newTime) {
                        newDate.setTime(newTime.getTime());
                        deferred.resolve(newDate);
                    }, function (err) {
                        // Handle error.
                        console.error(err);
                    });
                    }, function (err) {
                        // Handle error.
                        console.error(err);
                    });
            }, false);
```

**Comment by skwasjer** [comment]
This should now be fixed in v0.9.0.
